assets/scss/modules/_forms.scss

```diff
@@ -54,3 +54,29 @@
     transform: rotate(45deg);
   }
 }
+
+/* login form */
+
+.login-form {
+
+  .legend {
+    font-size: $dp-font-size-small;
+    float: right;
+    color: $dp-color-sylver;
+    font-style: italic;
+    font-weight: $dp-font-weight-light;
+  }
+
+  .forgot-link {
+    display: block;
+    margin: $dp-spaces--lv4 $dp-spaces--lv0;
+    color: $dp-color-green;
+    font-size: $dp-font-size-form;
+    font-style: italic;
+    font-family: $dp-font-family;
+
+      &:hover {
+        color: $dp-color-darkgreen;
```

Comment:
Why do we need a special hover color for a particular link? (Probably the same question applies for most of the `.forgot-link`'s properties)

Reply:
All links are green and dark green in hover.
Maybe we would have to change it in base.scss for all the links, and only add the classes for special cases.